app/components/sessions/forms/new-session.js

```diff
@@ -145,7 +145,7 @@ export default Component.extend({
       const kind = CONFIG.kinds.find((kind) => kind.uri === this.selectedKindUri);
       const postfix = (kind && kind.postfix) || '';
       this.set('selectedMainMeeting', mainMeeting);
-      this.set('startDate', mainMeeting.plannedstart);
+      this.set('startDate', mainMeeting.plannedStart);
```

Comment:
Een vergissing. Ik had de `closestMeeting.plannedStart` in `createAgendaitemToApproveMinutes` moeten terugzetten naar all lowercase, niet deze. Ik doe het meteen.

Reply:
ik snap je commentaar niet goed hier.
Ik had even snel dit stukje aangepast omdat we hier zijn gegaan naar all lowercase
Het attribuut `plannedStart` op meeting is camelcase dus dit is de correcte manier.
Wat bedoel je dan met `Ik had de closestMeeting.plannedStart moeten terugzetten naar all lowercase, niet deze`

Reply:
Zie de commit hieronder.
`agendaService.getClosestMeetingAndAgendaId(...)` returnt, verwarrend, een object met plannedstart in lowercase.